LayoutTests/http/wpt/webauthn/public-key-credential-get-failure-hid-silent.https.html

```diff
@@ -54,6 +54,6 @@
 
         if (window.internals)
             internals.setMockWebAuthenticationConfiguration({ silentFailure: true, hid: { stage: "request", subStage: "msg", error: "malicious-payload", canDowngrade: true, payloadBase64: [testCtapErrInvalidCredentialResponseBase64] } });
-        return promiseRejects(t, "NotAllowedError", navigator.credentials.get(options), "Operation timed out.");
+        return promiseRejects(t, "NotAllowedError", navigator.credentials.get(options), "Empty appid in create request.");
```

Comment:
I don't think this test expectation should be modified. We should make a new test passing in the empty string for appid and confirm it throws the right thing.